### PR TITLE
Full repo lint to apply new lint configurations for the sake of consistency.

### PR DESCRIPTION
**Pull request by PaulSonOfLars**
Apply all new lint rules for a consistent set of YAML

### CODE REVIEW

File: .yamllint.yaml [hunk 1]
```diff
@@ -11,6 +11,11 @@ yaml-files:
   - OWNERS
 
 rules:
+  braces:
+    min-spaces-inside: 1
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
```
Comment (PaulSonOfLars) [inline .yamllint.yaml]: The only functional change to this PR was adding this block, which makes yamllint agree with yamlfmt on the formatting of the braces in `examples/site.yml`

File: tasks/install-on-Windows.yml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ---
 - name: make password for user
   set_fact:
-    buildkite_agent_user_password: "{{ lookup('password', '/tmp/bk-agent-password length=32 chars=ascii_letters,digits,punctuation') }}"
-  no_log: "{{ buildkite_agent_hide_secrets | default(true) }}"
+    buildkite_agent_user_password: '{{ lookup(''password'', ''/tmp/bk-agent-password length=32 chars=ascii_letters,digits,punctuation'') }}'
```
Comment (DoomGerbil) [inline tasks/install-on-Windows.yml]: This line seems iffy to me - the double single-quotes seem wrong.

Does this pass lint and work?

File: tasks/install-on-Windows.yml [hunk 3]
```diff
@@ -89,27 +89,27 @@
...
 
 - name: configure service
   win_service:
     name: buildkite-agent
-    username: "{{ buildkite_agent_username }}"
-    password: "{{ buildkite_agent_user_password }}"
-    state: "{{ buildkite_agent_allow_service_startup[ansible_os_family] | ternary('started', 'stopped') }}"
+    username: '{{ buildkite_agent_username }}'
+    password: '{{ buildkite_agent_user_password }}'
+    state: '{{ buildkite_agent_allow_service_startup[ansible_os_family] | ternary(''started'', ''stopped'') }}'
```
Comment (DoomGerbil) [inline tasks/install-on-Windows.yml]: Ditto this line on the double single-quotes.

File: tasks/install-on-Debian.yml [hunk 4]
```diff
@@ -56,21 +56,21 @@
...
+    mode: '0644'
   notify:
     - restart-systemd-buildkite
 
 - name: Create buildkite-agent service instance
   systemd:
-    name: "buildkite-agent"
-    state: "{{ buildkite_agent_allow_service_startup[ansible_os_family] | ternary('started', 'stopped') }}"
+    name: buildkite-agent
+    state: '{{ buildkite_agent_allow_service_startup[ansible_os_family] | ternary(''started'', ''stopped'') }}'
```
Comment (DoomGerbil) [inline tasks/install-on-Debian.yml]: Same question about double single-quotes.

**Review by DoomGerbil — APPROVED** [review]
Talked offline about some weird constructions, but otherwise LGTM

File: tasks/install-on-Windows.yml (hunk 2 above)
Comment (bombsimon) [reply tasks/install-on-Windows.yml]: This is the YAML way to escape `'`. Even though one might think `\'` would be the way to do it, it's not. [Reference](https://yaml.org/spec/current.html#id2534365)

**Pull request merged**